config/datasources.yml

```diff
@@ -1,13 +1,28 @@
 apiVersion: 1
 
 datasources:
-- name: SystemLink Notebooks
-  type: systemlink-notebook-datasource
-  access: direct
-  url: /
-  version: 1
-- name: SystemLink Test Monitor
-  type: ni-systemlink-test-monitor
-  access: direct
-  url: /
-  version: 1
\ No newline at end of file
+  - name: SystemLink Notebooks
+    type: systemlink-notebook-datasource
+    uid: notebook
```

Comment:
Probably worth verifying, but I think in our testing adding a uid to an existing data source that has been provisioned and used in dashboards caused Grafana to see it as a new data source (since it generates a uid otherwise), and breaks dashboards.

Reply:
That doesn't seem to be the case in the interactive testing we've done, but I also don't claim a fully tested upgrade scenario. That workflow is blocked by at least this issue: #50  

I'd prefer not to pull that effort in to this PR since the issue is technically pre-existing in my mind;

Reply:
Sure I wouldn't expect that fix to be included in this PR. But I would at least ensure that adding the uid to existing data sources doesn't break dashboards that reference them, because that would likely be a major issue for customers.

Reply:
I fixed this.